tests/functional/workflows/python_pip/test_packager.py

```diff
@@ -449,6 +449,48 @@ def test_can_get_whls_mixed_compat(self, tmpdir, osutils, pip_runner):
         for req in reqs:
             assert req in installed_packages
 
+    def test_can_support_pep_600_tags(self, tmpdir, osutils, pip_runner):
+        reqs = ["foo"]
+        pip, runner = pip_runner
+        appdir, builder = self._make_appdir_and_dependency_builder(reqs, tmpdir, runner)
+        requirements_file = os.path.join(appdir, "requirements.txt")
+        pip.packages_to_download(
+            expected_args=["-r", requirements_file, "--dest", mock.ANY, "--exists-action", "i"],
+            packages=[
+                "foo-1.2-cp36-cp36m-manylinux_2_12_x86_64.whl",
+            ],
+        )
+
+        site_packages = os.path.join(appdir, ".chalice.", "site-packages")
+        with osutils.tempdir() as scratch_dir:
+            builder.build_site_packages(requirements_file, site_packages, scratch_dir)
+        installed_packages = os.listdir(site_packages)
+
+        pip.validate()
+        for req in reqs:
+            assert req in installed_packages
+
+    def test_can_support_compressed_tags(self, tmpdir, osutils, pip_runner):
+        reqs = ["foo"]
+        pip, runner = pip_runner
+        appdir, builder = self._make_appdir_and_dependency_builder(reqs, tmpdir, runner)
+        requirements_file = os.path.join(appdir, "requirements.txt")
+        pip.packages_to_download(
+            expected_args=["-r", requirements_file, "--dest", mock.ANY, "--exists-action", "i"],
+            packages=[
+                "foo-1.2-cp36-cp36m-manylinux_2_5_x86_64.manylinux1_x86_64.whl",
```

Comment:
> each tag in a filename can instead be a '.'-separated, sorted, set of tags

Can you double check whether this is sorted?

1. manylinux_2_5_x86_64
2. manylinux1_x86_64

Reply:
(and `manylinux1_x86_64` is an alias of `manylinux_2_5_x86_64`, they are the same right)